test/groovy/DockerExecuteOnKubernetesTest.groovy

```diff
@@ -64,22 +67,29 @@ class DockerExecuteOnKubernetesTest extends BasePiperTest {
         portList = []
         containerCommands = []
         bodyExecuted = false
-        JenkinsUtils.metaClass.static.isPluginActive = { def s -> new PluginMock(s).isActive() }
-        helper.registerAllowedMethod('sh', [Map.class], {return whichDockerReturnValue})
-        helper.registerAllowedMethod('container', [Map.class, Closure.class], { Map config, Closure body -> container(config){body()}
+        JenkinsUtils.metaClass.static.isPluginActive = { def s ->
```

Comment:
Formatting change only. Same code was present some lines above also earlier. I would like to suggest not to apply such formatting change in order not to blur the focus of that PR.